src/unminify.ts

```diff
@@ -17,6 +17,12 @@ export async function unminify(
 
     const file = extractedFiles[i];
     const code = await fs.readFile(file.path, "utf-8");
+
+    if (code.trim().length === 0) {
+      verbose.log(`Skipping empty file ${file.path}`);
+      continue;
+    }
```

Comment:
> @0xdevalias thanks for pointing this out, added a commit to verify that there's no extra processing on the top level
> 
> _Originally posted by @jehna in https://github.com/jehna/humanify/issues/134#issuecomment-2395585395_

@jehna This was a suggestion I made only as a quick/dirty hacky workaround. By skipping files at the `unminify` level you technically weaken the flexibility of the plugins; as it means a plugin can no longer take potentially empty code as input, and have valid code as output. While that's technically not an issue with how things are currently implemented; the other solutions I mentioned would solve this better/more flexibly.

- https://github.com/jehna/humanify/issues/54#issuecomment-2381073944